### DISCUSSION TITLE
Github developer pack

### DISCUSSION THREAD

**Discussion by vishwaraj14** [github/india#734] [GitHub Platform]
Update your Microsoft Azure skills on LinkedIn to claim GitHub Student Developer Pack.
I have not learn Azure skill   . How I can upadte ?

**Comment by prathikshetty2002** [comment]
To claim the GitHub student developer pack you don't need to learn it . 
Go to https://education.github.com/pack and apply for one using your college id or any other details listed. It usually is approved in 2-3 days or 1 week.
